images/dptp-3312-hypershift-leaks-cleaner/Dockerfile

```diff
@@ -1,7 +1,14 @@
+FROM registry.ci.openshift.org/ci/hypershift-cli:latest AS hypershift
+FROM registry.ci.openshift.org/ocp/4.14:cli AS oc
```

Comment:
Those multi-stage builds can be achieved in ci-op's config. You will see examples there.